db/schema.rb

```diff
@@ -61,4 +61,4 @@
   add_foreign_key "kudos_to_receivers", "users", column: "receiver_id"
   add_foreign_key "likes", "kudos"
   add_foreign_key "likes", "users", column: "giver_id"
-end
+end
```

Comment:
newline issue here too.

Reply:
hmm i have it on my computer

Reply:
oh jk, I think I made the comment before your prior commit.